### PR TITLE
Prepare preprod For 116-rc2

### PR DESCRIPTION
**Pull request by jnels124**
* prepare preprod for 116-rc2

### CODE REVIEW

**Review by edwin-greene — APPROVED** [review]

File: clusters/preprod/common/helmrelease.yaml [hunk 1]
```diff
@@ -13,7 +13,7 @@ spec:
       valuesFiles:
         - values.yaml
         - values-v2.yaml
-      version: '0.116.0-rc1'
+      version: '0.116.0-rc2'
```
Comment (steven-sheehy) [inline clusters/preprod/common/helmrelease.yaml:16]: We should switch this back to wildcard

File: clusters/preprod/common/helmrelease.yaml [hunk 2]
```diff
@@ -43,6 +43,8 @@ spec:
         resources:
           limits:
             memory: 512Mi
+    minio:
```
Comment (steven-sheehy) [inline clusters/preprod/common/helmrelease.yaml:46]: Shouldn't we have backups in all public envs?

File: clusters/preprod/common/helmrelease.yaml (hunk 1 above)
Comment (steven-sheehy) [reply clusters/preprod/common/helmrelease.yaml:16]: Nevermind, for some reason thought this was previewnet

**Review by steven-sheehy — APPROVED** [review]

**Comment by steven-sheehy** [comment]
Should we also update perf1?

**Review by steven-sheehy — APPROVED** [review]

**Review by edwin-greene — APPROVED** [review]

**Pull request merged**